joomla/joomla-cms
# [4.0] Improve card layout for JED extensions

Pull Request for Issue #30138 .

### Summary of Changes
This is somehow tricky as the content of this panel is delivered from the JED.

I added a own class to the whole panel. This enables styling of the cards in a way that they fit better. 
This means we now have a new file com_installer.scss.

- I removed the top border radius from the images
- Force the images to height 5rem. This meand that huge images are compressed - this is not beautiful, but huge images are not good either
- Reduced the padding around texts 
- Made more top margin between cards
- Hide overrides 

Please note: This does NOT resolve problems of this view on small devices, as the output is generated (as row instead of row-fluid) somewhere ..  . 

### Testing Instructions
Look a at the webinstaller tab. You see some cards with big images where the text is outside the cards.
Apply the patch and run npm
Look again, all images have same height  and nothing overlaps the card borders.


### Actual result BEFORE applying this Pull Request
See the issue #30138


### Expected result AFTER applying this Pull Request
<img width="778" alt="install-from-web" src="https://user-images.githubusercontent.com/1035262/87884268-9c118180-ca0d-11ea-843d-9a80bdba4abc.PNG">



### Documentation Changes Required
no
